app/pages/home-for-user/index.jsx

```diff
@@ -202,7 +212,16 @@ const HomePageForUser = React.createClass({
             <div>{this.state.error.toString()}</div>
           )}
 
-          {!!hashQuery.project ? (
+          {screen.width > 700 && (
```

Comment:
Is the intention to hide the button if the window is less than 700px? If so, then use `window.innerWidth` instead of screen. Right now, if I drag my browser to be smaller than 700px, the button is still visible.